starboard/build/config/modular/shared/BUILD.gn

```diff
@@ -0,0 +1,224 @@
+    "__ELF__",
+
+    # Use scalar portable implementations instead of Clang/GCC vector
+    # extensions in SkVx.h.
+    "SKNX_NO_SIMD",
+
+    # By default, <EGL/eglplatform.h> pulls in some X11 headers that have some
+    # nasty macros (|Status|, for example) that conflict with Chromium base.
+    "MESA_EGL_NO_X11_HEADERS",
+  ]
+
+  if (is_debug) {
+    cflags += [
+      "-O0",
+      "-frtti",
+    ]
+    if (!(is_host_win && using_old_compiler)) {
+      cflags += [
+        # This flag causes significant increase in shared library binary size on certain windows platforms. Refer b/297357707
+        "-g",
+      ]
+    }
+  } else if (is_devel) {
+    cflags += [
+      "-O2",
+      "-frtti",
+    ]
+    if (!(is_host_win && using_old_compiler)) {
+      cflags += [
+        # This flag causes significant increase in shared library binary size on certain windows platforms. Refer b/297357707
+        "-g",
+      ]
+    }
+  } else {
+    cflags += [ "-fno-rtti" ]
+    if (!(is_host_win && using_old_compiler)) {
+      cflags += [
+        # This flag causes significant increase in shared library binary size on certain windows platforms. Refer b/297357707
+        "-gline-tables-only",
+      ]
+    }
+  }
+
+  if (is_clang) {
+    cflags += [
+      "-fcolor-diagnostics",
+
+      # Default visibility to hidden, to enable dead stripping.
+      "-fvisibility=hidden",
+
+      # Warns on switches on enums that cover all enum values but
+      # also contain a default: branch. Chrome is full of that.
+      "-Wno-covered-switch-default",
+
+      # protobuf uses hash_map.
+      "-Wno-deprecated",
+
+      "-fno-exceptions",
+
+      # Enable unwind tables used by libunwind for stack traces.
+      "-funwind-tables",
+
+      # Disable usage of frame pointers.
+      "-fomit-frame-pointer",
+
+      # Don"t warn about the "struct foo f = {0};" initialization pattern.
+      "-Wno-missing-field-initializers",
+
+      # Do not warn for implicit sign conversions.
+      "-Wno-sign-conversion",
+
+      "-fno-strict-aliasing",  # See http://crbug.com/32204
+
+      "-Wno-unnamed-type-template-args",
+
+      # Triggered by the COMPILE_ASSERT macro.
+      "-Wno-unused-local-typedef",
+
+      # Do not warn if a function or variable cannot be implicitly
+      # instantiated.
+      "-Wno-undefined-var-template",
+
+      # Do not warn about an implicit exception spec mismatch.
+      "-Wno-implicit-exception-spec-mismatch",
+
+      # It's OK not to use some input parameters.
+      "-Wno-unused-parameter",
+      "-Wno-conversion",
+      "-Wno-bitwise-op-parentheses",
+      "-Wno-shift-op-parentheses",
+      "-Wno-shorten-64-to-32",
+      "-fno-use-cxa-atexit",
+    ]
+  }
+
+  if (is_clang_16) {
+    cflags += [
+      # Do not remove null pointer checks.
+      "-fno-delete-null-pointer-checks",
+    ]
+  }
+
+  if (use_asan) {
+    cflags += [
+      "-fsanitize=address",
+      "-fno-omit-frame-pointer",
+    ]
+
+    ldflags = [
```

Comment:
Defining ldflags in this shared config seems strange to me unless these are truly shared across all linkers

Reply:
yea it's somewhat strange it worked for ps4 , so i thought ill keep it.

Ill try removing the ldflags and see what happens

Reply:
removed and ps4 still passes - ill remove these